common/grouping/grouping-en.html

```diff
@@ -0,0 +1,252 @@
+---
+{
+	"title": "Grouping Content - Scaffolding",
+	"language": "en",
+	"altLangPage": "grouping-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-28"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
```

Comment:
Considering we are trying to demo all scaffolding grouping element and this plugin do interfere with the plain `<pre>` element. Instead of applying globally, apply the prettify only on the code sample element. See the docs here: https://wet-boew.github.io/wet-boew/demos/prettify/prettify-en.html

```suggestion
```